### PR TITLE
Submitting PDFCreator Terminal Server version 5.2.0

### PR DESCRIPTION
**Pull request by frank-heindoerfer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132862)

### CODE REVIEW

**Comment by frank-heindoerfer** [comment]
> @frank-heindoerfer please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Avanquest pdfforge GmbH"

File: manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: Avanquestpdfforge.PDFCreator-Terminal-Server
+PackageVersion: 5.2.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /COMPONENTS="MAINPROGRAM" /SILENT /DEFERLICENSECHECK
```
Comment (stephengillie) [inline manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml:10]: Which license check is deferred with the switch `DEFERLICENSECHECK` ("Defer License Check")?

Comment (karlo-mirkovic) [reply manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml:10]: Hi @stephengillie,

I am a colleague of @frank-heindoerfer and part of the development team of PDFCreator. I will try to provide you with some more information.

The `DEFERLICENSECHECK` parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer. 

The parameter is necessary here in order to allow our licensed software to be installed with a simple `winget install` command (without additional parameters).

**Comment by karlo-mirkovic** [comment]
Hi @stephengillie,

I am a colleague of @frank-heindoerfer and part of the development team of PDFCreator. I will try to provide you with some more information.

The `DEFERLICENSECHECK` parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer. 

The parameter is necessary here in order to allow our licensed software to be installed with a simple `winget install` command (without additional parameters).

File: manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml (hunk 1 above)
Comment (frank-heindoerfer) [reply manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml:10]: Hello @stephengillie,

@karlo-mirkovic  has explained the function /DEFERLICENSECHECK very well and I have nothing to add.

>The DEFERLICENSECHECK parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer.

>The parameter is necessary here in order to allow our licensed software to be installed with a simple winget install command (without additional parameters).

**Comment by stephengillie** [comment]
> Hi @ stephengillie,
> 
> I am a colleague of @frank-heindoerfer and part of the development team of PDFCreator. I will try to provide you with some more information.
> 
> The `DEFERLICENSECHECK` parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer.
> 
> The parameter is necessary here in order to allow our licensed software to be installed with a simple `winget install` command (without additional parameters).

Hi @karlo-mirkovic,

Thanks for your thorough reply. This is fine, as a mode to allow unattended installation. Our concern is preventing automatic acceptance of EULAs during install, as most EULAs do not support this and instead require explicit user agreement. 

But this isn't occurring here and we can proceed as normal.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**